scripts/newRule.js

```diff
@@ -61,10 +62,18 @@ const testTemplate =
 
 const projectDir = path.dirname(__dirname);
 
-const rulePath = path.join(projectDir, `./src/rules/${ruleCamelName}Rule.ts`);
-fs.writeFileSync(rulePath, ruleTemplate)
+const newRulePath = `./src/rules/${ruleCamelName}Rule.ts`;
+fs.access(newRulePath, fs.F_OK, (err) => {
```

Comment:
Instead of this check, we should just be able to use [`wx+` mode](https://nodejs.org/api/fs.html#fs_fs_open_path_flags_mode_callback) for our writes I think